website_seo_redirection/models/website_seo_redirection.py

```diff
@@ -15,7 +17,9 @@ class WebsiteSeoRedirection(models.Model):
     _description = "SEO controller redirections"
     _sql_constraints = [
         ("origin_unique", "UNIQUE(origin)", "Duplicated original URL"),
-        ("destination_unique", "UNIQUE(destination)", "Duplicated new URL"),
+        ("origin_destination_distinct",
+         "CHECK(origin != destination)",
```

Comment:
Hmmm what about a cyclical recursion? Redirect A points to B which points to C, which points back at A? Out of scope?